acir/src/circuit/directives.rs

```diff
@@ -159,17 +162,23 @@ impl Directive {
                     write_u32(&mut writer, *i)?;
                 }
             }
-            Directive::Log(info) => match info {
-                LogInfo::FinalizedOutput(output_string) => {
-                    write_bytes(&mut writer, output_string.as_bytes())?;
-                }
-                LogInfo::WitnessOutput(witnesses) => {
-                    write_u32(&mut writer, witnesses.len() as u32)?;
-                    for w in witnesses {
-                        write_u32(&mut writer, w.witness_index())?;
+            Directive::Log {
+                is_trace,
+                output_info,
+            } => {
```

Comment:
This formatting seems odd to me. Apparently we never copied over the same `.rustfmt.toml` file to this repo when it was split out of noir.